Automattic/wp-calypso
# Design Picker: Switch to `react-intersection-observer` (v2)

See https://github.com/Automattic/wp-calypso/issues/74022
Previously https://github.com/Automattic/wp-calypso/pull/71064 (and https://github.com/Automattic/wp-calypso/pull/75594)

## Proposed Changes

Switches to `react-intersection-observer` in the Design Picker because `calypso/lib/use-in-view` is buggy and it doesn't make sense to spend time reinventing the wheel.

## Testing Instructions

1. Create a new Premium site.
2. Hit `Continue` until you get to the Design Picker.
3. Click on a category and verify `calypso_signup_design_scrolled_to_end` is fired when you hit the end of the page.

<img width="1715" alt="image" src="https://user-images.githubusercontent.com/36432/234577858-0e446ac3-d753-421c-bdf8-66454425c8b8.png">
